app.js

```diff
@@ -1,80 +1,88 @@
-// JS for booklist
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
```

Comment:
- [ ] [Optional] I would also advise you use unique id property for each book since it would make it easier to remove books with duplicate values.